### PR TITLE
Feat: add Multi-line and multi-cursor editing

### PR DESCRIPTION
**Pull request by alexVasylenko**
Hi! We want to add the next functionality https://kencenerelli.wordpress.com/2018/03/25/visual-studio-code-multi-line-and-multi-cursor-editing/ .

So, a user can edit multiple lines at the same time and a user can set multiple selections.

### CODE REVIEW

**Comment by czgu** [comment]
Hey Alex, this feature is already supported in Querybook. The shortcut to trigger it is cmd+click

**Comment by alexVasylenko** [comment]
> Hey Alex, this feature is already supported in Querybook. The shortcut to trigger it is cmd+click

Hi! Multi-Cursor Editing works, but this commit is only for Multi-Line Editing. I understand, that a user can get the Multi-line editing using cmd+click, but the typical combination for this is ctrl+alt+arrow and a lot of users know this combination (I checked it in VSCode and sublime editors).

File: querybook/webapp/const/keyMap.ts [hunk 1]
```diff
@@ -120,6 +120,14 @@ const DEFAULT_KEY_MAP = {
             key: 'Alt',
             name: 'Alt-1, Alt-2, ..., Alt-9 to change the engine to the nth on the engine list.',
         },
+        addCursorToPrevLine: {
+            key: 'Cmd-Alt-Up',
+            name: '',
```
Comment (czgu) [inline querybook/webapp/const/keyMap.ts:125]: add explanation of what it does here

**Comment by czgu** [comment]
@alexVasylenko ah that makes more sense now, just a minor comment

**Comment by alexVasylenko** [comment]
> 

Done!

**Comment by czgu** [comment]
@alexVasylenko I took out the shortcut in the description since they will be shown with the key in this UI 
image

lmk if you want to make any other changes, otherwise lgtm

**Review by czgu — APPROVED** [review]

**Pull request merged**